--- templates/tiki.tpl	2010-12-09 09:33:26.000000000 -0200
+++ templates/tiki.tpl	2011-08-11 11:18:41.557759670 -0300
@@ -96,7 +96,7 @@
 
 {if $prefs.feature_fullscreen != 'y' or $smarty.session.fullscreen != 'y'}
 					<hr class="hidden" />{* for semantic separation of center and side columns *}
-	{if  $prefs.feature_left_column eq 'fixed' or ($prefs.feature_left_column ne 'n' && $left_modules|@count > 0 && $show_columns.left_modules ne 'n')}
+	{if ($prefs.feature_left_column eq 'fixed' or ($prefs.feature_left_column ne 'n' && $left_modules|@count > 0 && $show_columns.left_modules ne 'n')) && ($prefs.language eq 'pt-br' || $tiki_p_admin eq 'y')}
 					<div id="col2"{if $prefs.feature_left_column eq 'user'} style="display:{if isset($cookie.show_col2) and $cookie.show_col2 ne 'y'} none{elseif isset($ie6)} block{else} table-cell{/if};"{/if}{if $prefs.feature_bidi eq 'y'} dir="rtl"{/if}>
 						<h2 class="hidden">Sidebar</h2>
 						<div class="content">
