--- templates/tiki-view_articles.tpl	2011-09-13 08:15:23.646208269 -0300
+++ templates/tiki-view_articles.tpl	2011-11-07 11:20:05.858616829 -0200
@@ -1,3 +1,13 @@
+{if $type == 'noticia'}
+	{assign var=title value='Notícias'}
+{elseif $type == 'editorial'}
+	{assign var=title value='Editoriais'}
+{elseif $type == 'revista'}
+	{assign var=title value='Edições da revista'}
+{elseif $type == 'artigo'}
+	{assign var=title value='Artigos'}
+{/if}
+
 {if !isset($actions) or $actions eq "y"}
 	{if $prefs.art_home_title ne ''}
 		{title help="Articles" admpage="cms"}
