--- tiki-read_article.php	2008-09-30 16:07:08.000000000 -0300
+++ tiki-read_article.php	2008-09-30 16:32:51.000000000 -0300
@@ -24,6 +24,9 @@
 
 $article_data = $tikilib->get_article($_REQUEST["articleId"]);
 
+/* gambi para funcionar o highlight do menu lateral */
+$smarty->assign('nomepagina', $article_data['type']);
+
 if ($article_data === false) {
 	$smarty->assign('errortype', 401);
 	$smarty->assign('msg', tra('Permission denied'));
