--- tiki-module_controls.php	2010-08-03 19:40:35.000000000 -0300
+++ tiki-module_controls.php	2011-08-11 11:23:03.967759790 -0300
@@ -44,5 +44,6 @@
 }
 // Fix locaton if parameter was removed...
 if ($url != $_SERVER["REQUEST_URI"]) header('location: ' . $url);
+include_once('gambi_nomes.php');
 $smarty->assign('current_location', $url);
 $smarty->assign('mpchar', (strpos($url, '?') ? '&' : '?'));
