--- webcalendar/includes/init.php	2008-12-02 23:06:29.000000000 -0200
+++ webcalendar/includes/init.php	2008-12-02 23:07:48.000000000 -0200
@@ -39,6 +39,12 @@
  *
  */
 
+/* gambi para o webcalendar pegar o mesmo login do tiki */
+session_start();
+if (empty($_SESSION['u_info']['login']) && $_SESSION['u_info']['groups'] != 'Admins') {
+    header('Location: ../tiki-login.php');
+}
+
 // Security Check
 if ( empty ( $PHP_SELF ) && ! empty ( $_SERVER ) &&
   ! empty ( $_SERVER['PHP_SELF'] ) ) {
